app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,15 @@
+from .dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str, skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.skill_level = skill_level

```

Comment:
The `skill_level` attribute should be a protected attribute. Consider renaming it to `_skill_level` to follow encapsulation best practices.